src/components/Footer/footer.scss

```diff
@@ -0,0 +1,202 @@
+@import "src/styles/common/variables";
+@import "src/styles/common/functions";
+@import "src/styles/common/mixin";
+
+@layer component {
+    .footer {
+        &__wrapper {
+            position: relative;
+            padding-top: rem(177);
```

Comment:
what means rem(177) ?

Reply:
This feature is necessary to make it easier to translate pixels into rem. Also, if we want to increase or decrease all fonts, it will be enough to change only font-size in the $html-font-size variable.

Reply:
For me it is unclear how it works. padding-top: rem(177); for me means nothing rather than padding-top: 15px

Reply:
rem is a relative unit relative to the font size of the root element (html).
For example we have font-size for html = 10px, then in this case all sizes that we convert to rem should be divided by 10. i.e. if I set the padding to 177 px, then it will be 17.7rem. But if I have for example font-size for html = 12px, then it is not so easy to translate pixels into rem every time. For this purpose there is a function to which we pass the value in pixels rem(177) https://prnt.sc/JU6UyiPTJgtY =>(177/12) +rem. This is also convenient when we want to change the scale of the layout depending on the resolution, for example, on a tablet the layout looks too large or on the contrary small, and for this purpose it will be enough for me to change the font-size in html on certain resolutions and the scale of the project will either decrease or increase. Also rem units work better with scaling. To summarize, this is just a function that allows you to convert pixels to rem in a simpler way. And using rem in projects is a good practice for a better product that will be easier to adapt to future changes.